### PR TITLE
CHANGELOG: Update for 25.0 release

### PR DESCRIPTION
**Pull request by pratheekshasn**
Updated the CHANGELOG file for the 25.0 scarthgap release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,10 +45,36 @@ Branch: `nilrt/master/next`
...
+- [Removed](https://github.com/ni/meta-nilrt/pull/766) the ni-wireless-cert* IPKs from images.
+- [Disabled](https://github.com/ni/meta-nilrt/pull/767) DOCKER_BUILDKIT builds of images.
+
 
 #### Fixed
 - [Fixed](https://github.com/ni/meta-nilrt/pull/676) a parsing error in the recovery media provisioning scripts.
+- [Fixed](https://github.com/ni/meta-nilrt/pull/750) the grep warnings getting thrown on boot.
+- [Fixed]() sysconfig settings to prevent users from accidentally enabling the UI in the SNAC configuration.
+
+-----remove below later
```
Comment (chaitu236) [inline CHANGELOG.md]: ??

File: CHANGELOG.md [hunk 2]
```diff
@@ -366,6 +402,19 @@ Branch: `nilrt/22.5/hardknott`
...
+## 8.20
+Branch: `nilrt/25.0/sumo`
+
+### nilrt
+
+No changes.
+
+### meta-nilrt
+
+- [Added](https://github.com/ni/meta-nilrt/commit/c5aca13f9f69170f0b21b05ea5adde77253f7279) new signing key into opkg-keyrings.
```
Comment (chaitu236) [inline CHANGELOG.md]: Why commit url instead of PR like others?

File: CHANGELOG.md [hunk 3]
```diff
@@ -366,6 +402,19 @@ Branch: `nilrt/22.5/hardknott`
 - [Removed](https://github.com/ni/meta-nilrt/pull/290) packages dropped from upstream.
 
 
+----
+## 8.20
```
Comment (chaitu236) [inline CHANGELOG.md]: This is 8.22 release. Please add "No changes." for 8.20 and 8.21 - refer 8.19 for how to do this.

File: CHANGELOG.md (hunk 2 above)
Comment (chaitu236) [reply CHANGELOG.md]: You used text ".. a new signing key to the base system image" for scarthgap.. why not use same for here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,7 +16,11 @@ The evergreen, canonical changelog for *all NILRT branches* can be [found here](
 
 ----
 ## 11.0
-Branch: `nilrt/master/next`
+Branch: `nilrt/25.0/scarthgap`
+
+### nilrt
+#### Added
+- [Added](https://github.com/ni/nilrt/pull/293) a way to handle new fields in the CVE files.
```
Comment (chaitu236) [inline CHANGELOG.md]: No need to mention this as this is dev script mainly intended for internal NI use.

File: CHANGELOG.md [hunk 5]
```diff
@@ -25,6 +29,12 @@ Branch: `nilrt/master/next`
 - [Added](https://github.com/ni/meta-nilrt/pull/717) `libpwquality` to the core packagefeed.
 - [Added](https://github.com/ni/meta-nilrt/pull/726) `nftables` to runmode.
 - [Added](https://github.com/ni/meta-nilrt/pull/729) `nilrt-snac` recipe to put system into SNAC configuration.
+- [Added](https://github.com/ni/meta-nilrt/pull/732) onboard virtual keyboard for use in touch-screen devices.
```
Comment (chaitu236) [inline CHANGELOG.md]: From customer's perspective, onboard was always present in our released BSIs. The fact that it was removed in scarthgap and later add is irrelevant. So no need to mention this.

File: CHANGELOG.md [hunk 6]
```diff
@@ -25,6 +29,12 @@ Branch: `nilrt/master/next`
...
+- [Enabled](https://github.com/ni/meta-nilrt/pull/743) gnuradio.
```
Comment (chaitu236) [inline CHANGELOG.md]: Same here

File: CHANGELOG.md [hunk 7]
```diff
@@ -25,6 +29,12 @@ Branch: `nilrt/master/next`
...
+- [Added](https://github.com/ni/meta-nilrt/pull/764) sysconfig-settings-console subpackage that will allow removing the MAX toggle that can re-enable the consoleout disabled by the NILRT SNAC.
```
Comment (chaitu236) [inline CHANGELOG.md]: No need to mention this as well.

File: CHANGELOG.md [hunk 8]
```diff
@@ -35,10 +45,36 @@ Branch: `nilrt/master/next`
 - [Deprecated](https://github.com/ni/meta-nilrt/pull/663) the `ntpdate` package, since it has been dropped by upstream.
 - [Removed](https://github.com/ni/meta-nilrt/pull/673/commits/01ac041e848b67553001cee00336637eefb9384a) NIcurl-specific functions from curl, since these sources are no longer used by NIcurl.
 - [Removed](https://github.com/ni/meta-nilrt/pull/675) the unused `efifix` scripts.
+- [Removed](https://github.com/ni/meta-nilrt/pull/744) quagga entry from files group.
+- [Removed](https://github.com/ni/meta-nilrt/pull/749) sqlite3.
```
Comment (chaitu236) [inline CHANGELOG.md]: We only removed sqlite3 from extras feed as it's already in main feed. From customer's perspective, this isn't important. Remove

File: CHANGELOG.md (hunk 1 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: I'm so sorry, I thought I had removed that. Let me fix that.

File: CHANGELOG.md (hunk 2 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Because this one was a cherry-pick. Should I add the original PR instead?

File: CHANGELOG.md [hunk 9]
```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
 - [Deprecated](https://github.com/ni/meta-nilrt/pull/663) the `ntpdate` package, since it has been dropped by upstream.
 - [Removed](https://github.com/ni/meta-nilrt/pull/673/commits/01ac041e848b67553001cee00336637eefb9384a) NIcurl-specific functions from curl, since these sources are no longer used by NIcurl.
 - [Removed](https://github.com/ni/meta-nilrt/pull/675) the unused `efifix` scripts.
+- [Removed](https://github.com/ni/meta-nilrt/pull/744) quagga entry from files group.
```
Comment (chaitu236) [inline CHANGELOG.md]: Cleanup that's irrelevant to customers. Remove

File: CHANGELOG.md [hunk 10]
```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
...
+- [Removed](https://github.com/ni/meta-nilrt/pull/749) sqlite3.
+- [Removed](https://github.com/ni/meta-nilrt/pull/754) unused sudo access for lvuser.
+- [Removed](https://github.com/ni/meta-nilrt/pull/766) the ni-wireless-cert* IPKs from images.
+- [Disabled](https://github.com/ni/meta-nilrt/pull/767) DOCKER_BUILDKIT builds of images.
```
Comment (chaitu236) [inline CHANGELOG.md]: Irrelevant for customers. Remove

File: CHANGELOG.md [hunk 11]
```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
...
+- [Removed](https://github.com/ni/meta-nilrt/pull/769) deprecated skyline clients from the base system image.
+- [Removed](https://github.com/ni/meta-nilrt/pull/783) static linking of expat from libfmi.
```
Comment (chaitu236) [inline CHANGELOG.md]: This was broken in kirkstone->scarthgap upgrade which we fixed in this PR. So from customers' perspective, nothing changed. So remove this.

File: CHANGELOG.md [hunk 12]
```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
...
 
 #### Fixed
 - [Fixed](https://github.com/ni/meta-nilrt/pull/676) a parsing error in the recovery media provisioning scripts.
+- [Fixed](https://github.com/ni/meta-nilrt/pull/750) the grep warnings getting thrown on boot.
```
Comment (chaitu236) [inline CHANGELOG.md]: Just fixed a breakage in kirkstone->scarthgap upgrade. Not relevant for customers. Remove

File: CHANGELOG.md [hunk 13]
```diff
@@ -35,9 +47,21 @@ Branch: `nilrt/master/next`
...
+- [Fixed]() sysconfig settings to prevent users from accidentally enabling the UI in the SNAC configuration.
```
Comment (chaitu236) [inline CHANGELOG.md]: link?

Comment (chaitu236) [reply CHANGELOG.md]: No need to mention SNAC related changes I think unless it affects regular BSI as well.

File: CHANGELOG.md (hunk 2 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: I have added the PR link.

File: CHANGELOG.md (hunk 3 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 4 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 5 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 6 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 7 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 8 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 9 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 10 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 11 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 12 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 13 above)
Comment (pratheekshasn) [reply CHANGELOG.md]: Fixed.

**Comment by chaitu236** [comment]
Merged as 07329d1b46385521129b1afc288ee660122ed232 after minor modifications

**Pull request closed without merging**